modules/ROOT/pages/powerpaste-troubleshooting.adoc

```diff
@@ -20,18 +20,4 @@ Microsoft Word can also create content that cannot be accurately recreated in HT
 
 == How can I see what is on the clipboard?
 
-To view the HTML of content pasted from the clipboard:
-
-* If you are using Microsoft Internet Explorer 11, visit: http://static.ephox.com/clipboard/clipboardtestie11.html[{companyname} Clipboard Viewer for Microsoft Internet Explorer 11].
-* If you are using any other browser, visit: http://static.ephox.com/clipboard/clipboardtest.html[{companyname} Clipboard Viewer].
-
-== Why would Microsoft Internet Explorer 11 show different results from every other supported browser?
-
-Microsoft Internet Explorer interacts with Microsoft Word content differently than all other browsers. When pasting, Microsoft Internet Explorer transforms and cleans up Microsoft Word content before pasting it into {productname}. Therefore, Microsoft Internet Explorer provides different clipboard data to web applications when compared to other browsers. This behavior only occurs when Microsoft content is pasted into Microsoft Internet Explorer and cannot be disabled.
-
-== Why do images not paste when copied with text content in Microsoft Internet Explorer 11?
-
-This issue relates to changes to Microsoft Internet Explorer 11 late in the product’s life. {companyname} (and companies that offer similar products) have reached out to Microsoft to suggest that this change is a defect despite their initial reply that it was intentional and "expected behavior" in Microsoft Internet Explorer 11. Microsoft has made no public statement about addressing the issue specifically, and is no longer making non-security changes to Microsoft Internet Explorer 11. The only recommended workarounds are:
-
-* Paste the images into {productname} individually.
-* Use a different browser.
+To view the HTML of content pasted from the clipboard: http://static.ephox.com/clipboard/clipboardtest.html[{companyname} Clipboard Viewer].
```

Comment:
This doesn't read well... it says to view it and then just has a link (granted this was sort of a problem before as well). Maybe something like this?
```suggestion
To view the HTML of content on the clipboard paste into the red box on the http://static.ephox.com/clipboard/clipboardtest.html[{companyname} Clipboard Viewer].
```